.github/pull_request_template.md

```diff
@@ -14,10 +14,19 @@ and code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md
 -->
 
 - [ ] I have performed a self-review of my code.
-- [ ] I have added tests for my changes.
-- [ ] I have run linter locally (`make lint`) and all checks pass.
-- [ ] I have run tests locally (`make tests`) and all tests pass.
 - [ ] I have commented on my code, particularly in hard-to-understand areas.
+- [ ] I have added tests for my changes.
+- [ ] I have run linter and tests locally (`make lint` and `make tests`) and all pass.
+
+## Checklist: Documentation (only for k6 maintainers and if relevant)
+
+**Please do not merge this PR until the following items are filled out.**
```

Comment:
We can tweak it a bit so it's clear that some of these items are generally optional. I have no stats, but I can imagine a huge percentage of PRs we merge don't require any update on TypeScript definitions. Similar, but less often for k6-docs.